laws/shared/src/main/scala/scalaz/laws/HomomorphismLaws.scala

```diff
@@ -0,0 +1,191 @@
+package scalaz
+package laws
+
+import scala.{ Nothing, Unit }
+
+import data._, tc._, Scalaz._
+
+object HomomorphismLaws {
```

Comment:
why this and not `*Laws`s for the various implicated typeclasses?

Reply:
I believe this is from #1936. @alexknvl has suggested putting the laws into the type classes themselves which seems like an okay idea to me. I'll try that first.